interactive/dashboard/api.py

```diff
@@ -1002,3 +1002,15 @@ def post(self, request):
             request.session.create()
         logging.info("saving model options for user: "+request.session.session_key)
         uploaded = request.FILES['file']
+# checks server by trying to connect to default port
+def checkForRabbitMQServer():
```

Comment:
please convert this and all other functions to snake case, which is the standard python naming scheme

Reply:
Actually, delete this. When we try to connect to rabbit that connection will fail and we can just catch that exception

Reply:
removed